doc/plugins/prometheus.md

```diff
@@ -122,12 +122,56 @@ Or you can goto [Grafana official](https://grafana.com/grafana/dashboards/11719)
 
 ### Available metrics
 
-* `Status codes`: HTTP status codes returned by upstream services. These are available per service and across all services.
+* `Status`: HTTP status codes returned by upstream services. These are available per service and across all services.
+
+    Attributes：
+
+    | Name         | Description         |
+    | -------------| --------------------|
+    | code         | The HTTP status code returned by the upstream service. |
+    | route        | The route_id of the request to match the route. If it does not match, the default is `""`. |
+    | matched_uri  | Request to match the uri of the route. If it does not match, the default is `""`. |
+    | matched_host | Request to match the hsot of the route. If it does not match, the default is `""`. |
+    | service      | The request matches the service_id of the route. If it does not match, the default is host. |
+    | consumer     | The request matches the consumer_name of the consumer. If it does not match, the default is `""`. |
+    | node         | The request hits the ip of the upstream node. |
+
 * `Bandwidth`: Total Bandwidth (egress/ingress) flowing through apisix. This metric is available per service and as a sum across all services.
+
+    Attributes：
+
+    | Name         | Description |
+    | -------------| ------------- |
+    | type         | The type of bandwidth(egress/ingress). |
+    | route        | The route_id of the request to match the route. If it does not match, the default is `""`. |
+    | service      | The request matches the service_id of the route. If it does not match, the default is host. |
```

Comment:
The service_id of the route matched by the request?

Reply:
updated.